prototype/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## How to contribute
+
+Pocket is an open source project, and as such we welcome any contribution from anyone on the internet.
+
+Please fork, code and submit a Pull Request for the team to review and merge. We ask that you please follow the guidelines below in order to submit your contributions for review:
+
+### High impact or architectural changes
+
+Reach out to us on [Telegram](https://t.me/POKTnetwork) and start a discussion with the team regarding your change before you start working. Communication is key for open source projects and asynchronous contributions.
+
+For an active research forum, checkout and post on [our forum](https://research.pokt.network).
+
```

Comment:
I think discord can be used too, I think it's more practical than TG.

Reply:
Noted. This was just a transfer copy/paste of what we had before. Going to leave as is because we'll be deleting the `prototype` directory eventually and the newer instructions are in Notion.